src/chainstate/stacks/db/blocks.rs

```diff
@@ -11875,8 +12008,8 @@ pub mod test {
                 let chainstate = peer.chainstate();
                 let (mut chainstate_tx, clarity_instance) =
                     chainstate.chainstate_tx_begin().unwrap();
-                let (stack_stx_ops, transfer_stx_ops) =
-                    StacksChainState::get_stacking_and_transfer_burn_ops_v210(
+                let (stack_stx_ops, transfer_stx_ops, delegate_stx_ops) =
+                    StacksChainState::get_stacking_and_transfer_and_delegate_burn_ops_v210(
```

Comment:
Can you expand this test to cover expired delegate-stx on-Bitcoin operations?  Thanks!